tools/mpremote/mpremote/main.py

```diff
@@ -355,17 +357,23 @@ def load_user_config():
     # Create empty config object.
     config = __build_class__(lambda: None, "Config")()
     config.commands = {}
-
-    # Get config file name.
-    path = os.getenv("XDG_CONFIG_HOME")
-    if path is None:
-        path = os.getenv("HOME")
-        if path is None:
-            return config
-        path = os.path.join(path, ".config")
-    path = os.path.join(path, _PROG)
+    # use $XDG_CONFIG_HOME,$HOME $env:USERPROFILE% or $env:APPDATA
```

Comment:
That's pretty clear from the code below so I'd leave out this comment, or else use something like 'Look for configuration in common directories'